.gitignore

```diff
@@ -53,4 +53,6 @@ gams
 logging.conf
 logs
 turbine.cfg
-user_plugins
\ No newline at end of file
+user_plugins
+turbine_aws.cfg
+user_ml_ai_models/__init__.py
```

Comment:
Why was this added here?

Reply:
Because I accidentally added them in fa2d5b0e979ef915151bf6078ce25636f2a77272 !  I figured I (or others) might make the same mistake again so I added them to be ignored.